config/config_test.go

```diff
@@ -133,237 +133,6 @@ func (s *ConfigTestSuite) TestRuntimeCfgExists() {
 	}
 }
 
-func (s *ConfigTestSuite) TestConfigUpdate() {
```

Comment:
What is the reason for removing the tests here?

Reply:
The `Update` function is not being used anymore